restapi/header/header.avsc

```diff
@@ -4,8 +4,9 @@
   "name": "Header",
   "doc": "Metadata for Dataset. It is a summary explaining what kind of information are stored in the Dataset.",
   "fields": [
-    {"name": "descriptiveStatistic", "type": "DescriptiveStatistic" },
-    {"name": "unit", "type": "org.radarcns.avro.restapi.sensor.Unit"},
-    {"name": "effectiveTimeFrame", "type": "EffectiveTimeFrame" }
+    {"name": "descriptiveStatistic", "type": "DescriptiveStatistic", "doc": "Statical value expressed by samples" },
```

Comment:
It's more that when a query and the answer are not directly connected, such as with a data download or backup, or you have a call that returns data from multiple datasets, it is still possible to disambiguate what the dataset is describing. Plus, it would only be one-time fields in the header rather than repeating values in the dataset items.